packages/webapp/src/containers/Consent/locales/es/Owner.Consent.md

```diff
@@ -152,18 +156,18 @@ El sitio web de LiteFarm es de uso gratuito. El software en sí tiene la licenci
 
 Sobre la aplicación, comuníquese con:
 
-- Gerente de producto: Kevin Cussen (litefarm.team@ubc.ca)
+Gerente de producto: litefarm.team@ubc.ca 
 
-Sobre a este estudio, comuníquese con la investigadora principal:
+Sobre este estudio, comuníquese con la investigadora principal: 
 
-- Investigadora principal: Dra. Hannah Wittman (hannah.wittman@ubc.ca)
+Investigadora principal: Dra. Hannah Wittman ( hannah.wittman@ubc.ca )  
 
-**Si tiene algún comentario o queja sobre sus derechos como participante en la investigación y/o sus experiencias mientras participa en este estudio**
+Si tiene algún comentario o queja sobre sus derechos como participante en la investigación y/o sus experiencias mientras participan en este estudio  
 
-Comuníquese con la Línea de Quejas del(a) Participante de Investigación de la Oficina de Ética de Investigación de la UBC al 604-822-8598 (llamada local). Si la llamada es de larga distancia llame sin cargo al 1-877-822-8598, o envíe un correo electrónico a RSIL@ors.ubc.ca
+Comuníquese con la Línea de Quejas del(a) Participante de Investigación de la Oficina de Ética de Investigación de la UBC al 604-822-8598 (llamada local). Si la llamada es de larga distancia llame sin cargo al 1-877-822-8598, o envíe un correo electrónico a RSIL@ors.ubc.ca 
 
-**Consentimiento**
+Consentimiento   
```

Comment:
This will remain bolded in other languages.

Reply:
Thank you, let's fix that one, it will hinder reading to lose the section title.